modules/logging-loki-memberlist-ip.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="logging-loki-memberlist-ip_{context}"]
+= Configuring Loki to tolerate memberlist creation failure
+
+In an OpenShift cluster, the administrators generally use a non-private IP network range. As a result, the LokiStack memberlist configuration fails as, by default, it only uses private IP networks.
```

Comment:
```suggestion
In an OpenShift cluster, administrators generally use a non-private IP network range. As a result, the LokiStack memberlist configuration fails because, by default, it only uses private IP networks.
```

Reply:
Made this change. Thanks!